plan/plans/info.go

```diff
@@ -361,8 +361,14 @@ func (isp *InfoSchemaPlan) fetchColumns(schemas []*model.DBInfo) {
 				if decimal == types.UnspecifiedLength {
 					decimal = 0
 				}
-				dataType := types.TypeToStr(col.Tp, col.Charset == charset.CharsetBin)
-				columnType := fmt.Sprintf("%s(%d)", dataType, colLen)
+				columnType := types.TypeToStr(col.Tp, col.Charset == charset.CharsetBin)
+				if col.Decimal == types.UnspecifiedLength {
```

Comment:
I see other similar handing for type(m, n) in util/types and column, maybe we can unify them. 
Btw, you can see that float and double don't have (m, n) even they have length and decimal.